src/cuda-sim/ptx_sim.cc

```diff
@@ -202,7 +202,7 @@ unsigned ptx_thread_info::get_builtin(int builtin_id, unsigned dim_mod) {
     case CLOCK_REG:
       return (unsigned)(m_gpu->gpu_sim_cycle + m_gpu->gpu_tot_sim_cycle);
     case CLOCK64_REG:
-      abort();  // change return value to unsigned long long?
+      // abort();  // change return value to unsigned long long?
```

Comment:
Can we just remove instead of commenting, or add some context to the comment?
We would generally support a wider return type in the Ptx model, but this is a wider fix.

I guess in this instance, we will just return a truncated value?